dsfhub/resource_cloud_account.go

```diff
@@ -750,6 +747,10 @@ func resourceCloudAccountConnectionHash(v interface{}) int {
 		buf.WriteString(fmt.Sprintf("%s-", v.(string)))
 	}
 
+	if v, ok := m["auth_mechanism"]; ok {
+		buf.WriteString(fmt.Sprintf("%v-", v.(string)))
```

Comment:
not that it'll make a difference, but could we keep it a string-specific format `%s` rather than default format `%v` to avoid potential confusion or issues with type handling?